server/src/main/java/org/apache/cloudstack/backup/BackupManagerImpl.java

```diff
@@ -1022,31 +1018,23 @@ protected void runInContext() {
                         continue;
                     }
 
-                    // Sync backup usage metrics
                     final Map<VirtualMachine, Backup.Metric> metrics = backupProvider.getBackupMetrics(dataCenter.getId(), new ArrayList<>(vms));
-                    final GlobalLock syncBackupMetricsLock = GlobalLock.getInternLock("BackupSyncTask_metrics_zone_" + dataCenter.getId());
```

Comment:
Not locking now, how are the metrics update and backup sync affected by this?

Reply:
The locking may not be necessary now that we're publishing usage events so that the usage metrics updating is handled by usage server. I can put back the locking if necessary. 

Testing would be necessary.